sgx/src/lib.rs

```diff
@@ -78,7 +78,10 @@ pub mod crypto;
 
 #[cfg(feature = "std")]
 pub mod attestation_types;
+
 #[cfg(feature = "std")]
+// FIXME:
+#[allow(clippy::module_inception)]
```

Comment:
@npmccallum clippy does not like it:
https://rust-lang.github.io/rust-clippy/master/index.html#module_inception